src/spec/verifiable-payid-protocol-NC.md

```diff
@@ -0,0 +1,379 @@
+
+   Basic PayID protocol [PAYID-PROTOCOL][] specifies a protocol to interact with a PayID server and retrieve a payment account(s) address information resource along with other meta-data corresponding to the queried PayID. One of the security assumptions made by the basic PayID protocol that may be less desirable for some applications is that the owner of the PayID must trust their PayID server to provide correct and untampered responses. Under this model, the PayID server has full control over the contents of any PayID response message, with potentially adverse effects if the server goes rogue or is compromised. The PayID owner has no way of knowing if the PayID server behaves maliciously. This implicit trust assumption between the PayID owner and their PayID server is often unacceptable in a non-custodial setting.
+   
+   This protocol enables non-custodial service providers to provide non-custodial PayID service while preserving the existing trust assumptions between themselves and their users. More specifically, this protocol allows a PayID owner to digitally sign a PayID response using a local application/device with their PayID private key (which never leaves their device). This signed PayID response can then be securely transferred to the non-custodial PayID service provider's server who can then send this as a response to a PayID query along with PayID owner's "identity" information. PayID clients can use this information to verify if a PayID response is signed by the PayID owner, and then decide wheteher to proceed with any particular transaction.  
+
+# Verifiable PayID Protocol for Non-Custodial Services Specification
+  The Verifiable PayID protocol is designed along the same design principles as [PAYID-PROTOCOL][].
+
+## PaymentInformation Resource as JSON Web Signatures
+  The PayID Protocol [PAYID-PROTOCOL] defines a Payment Account(s) Information Resource that contains information about a particular PayID. This document further refines this definition to allow this information to be digitally signed, and then represented as a JSON Web Signature (JWS) [RFC7515] using JWS JSON Serialization.
+
+  Below, this document further defines the structure of each JWS component, for the purposes of Verifiable PayID.
+
+### JOSE Protected Header
+  For JWS, the members of the JSON object represented by the JOSE Header describe the cryptographic operations applied to the JWS Protected header and the JWS payload and optionally additional properties of the JWS.
+
+  For a complete list of members of this object, refer to [RFC7519][] and [RFC7515][]. Following is an example of a JWT object representing JOSE header parameters for JWS JSON Serialization syntax.
```

Comment:
```suggestion
  For a complete list of members of this object, refer to [RFC7515][]. Following is an example of a JWT object representing JOSE header parameters for JWS JSON Serialization syntax.
```

I may be missing something, but isn't everything about this defined in RFC7515 (JWS)? I suppose it's _actually_ defined in RFC-7517 (JWK) but it seems sufficient to only list 7515 here (?)

In any event, I don't think RFC-7519 (JWT) is involved here (correct me if I'm wrong though).

Reply:
RFC 7519 is a subset of RFC 7515. So I can remove that reference.